aidants_connect_web/templates/layouts/nav.html

```diff
@@ -9,26 +9,26 @@
       <ul class="nav__links">
       {% if request.user.is_authenticated %}
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'dashboard' %}">Espace Aidant</a>
+          <a {% if request.resolver_match.view_name in 'dashboard,usagers,usager_details,usagers_mandats_autorisations_cancel_confirm' %}class="font-weight-bold"{% endif %} href="{% url 'dashboard' %}">Mon espace Aidant</a>
         </li>
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'ressources' %}">Ressources</a>
+          <a {% if request.resolver_match.view_name in 'ressources' %}class="font-weight-bold"{% endif %} href="{% url 'ressources' %}">Ressources</a>
         </li>
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'logout' %}">Se déconnecter</a>
+          <a href="{% url 'logout' %}">Se déconnecter</a>
         </li>
       {% else %}
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'home_page' %}">Accueil</a>
+          <a {% if request.resolver_match.view_name == 'home_page' %}class="font-weight-bold"{% endif %} href="{% url 'home_page' %}">Accueil</a>
         </li>
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'ressources' %}">Ressources</a>
+          <a {% if request.resolver_match.view_name in 'ressources' %}class="font-weight-bold"{% endif %} href="{% url 'ressources' %}">Ressources</a>
         </li>
         <li class="nav__item">
-          <a class="button-outline secondary" href="{% url 'faq_generale' %}">FAQ</a>
+          <a {% if request.resolver_match.view_name|slice:':3' in 'faq' %}class="font-weight-bold"{% endif %} href="{% url 'faq_generale' %}">FAQ</a>
```

Comment:
Est-ce que `'faq' in view_name` ne serait pas une solution suffisante et plus lisible ? :)